flexmeasures/data/tests/test_sensor_queries.py

```diff
@@ -0,0 +1,19 @@
+from flexmeasures.data.services.resources import find_closest_sensor
+
+
+def test_closest_sensor(add_weather_sensors):
+    """Check that the closest temperature sensor to our wind sensor returns
+    the one that is on the same spot as the wind sensor itself.
```

Comment:
Is there another temperature further away? That would make this test better.

Or we make another one close b, test for n=1 and n=2